cns/service/main.go

```diff
@@ -856,20 +858,32 @@ func main() {
 			logger.Errorf("Failed to start multiTenantController, err:%v.\n", err)
 			return
 		}
+	}
 
+	logger.Printf("[Azure CNS] Start HTTP local echo server")
+	httpEchoRestService := restserver2.New(httpRestService)
```

Comment:
nit: instead of calling this `httpEchoRestService`, how about `localSrv`? since this is serving local requests, and then we can make the `httpRestService` to be `remoteSrv`

Reply:
maybe name httpRestService as httpLegacyService?
httpEchoRestService as httpLocalRestService?

Reply:
i like @nddq suggestion. Lets call it `httpLocalService` and `httpRemoteService`.